### PR TITLE
Allows you to see powersink power consumption

### PR DESCRIPTION
**Pull request by ThatLing**
Can on the tin. Using either a multitool or examining it while you're an observer will show you the amount of power the powersink has consumed.

:cl:  
rscadd: Using a multitool on a powersink will now show you the amount of power consumed.
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/powersink.dm [hunk 1]
```diff
@@ -78,6 +78,10 @@
 				"[user] detaches \the [src] from the cable.", \
 				"<span class='notice'>You detach \the [src] from the cable.</span>",
 				"<span class='italics'>You hear some wires being disconnected from something.</span>")
+	// yogs start - multitool for power drained
+	if(istype(I, /obj/item/multitool))
+		to_chat(user, "The power dial reads [power_drained] kW.")
```
Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [inline code/game/objects/items/devices/powersink.dm]: Shouldn't this be joules

File: yogstation/code/game/objects/items/devices/powersink.dm [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/obj/item/powersink/examine(mob/user)
+	. = ..()
+	if(isobserver(user))
+		to_chat(user, "The power dial reads [power_drained] kW.")
```
Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [inline yogstation/code/game/objects/items/devices/powersink.dm]: this too

**Review by 1fbff5f83b23d39d38b1dfcb4cac8d9b — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/powersink.dm (hunk 1 above)
Comment (ThatLing) [reply code/game/objects/items/devices/powersink.dm]: Using a multitool on a wire uses kW so I just used that unit for this.

File: yogstation/code/game/objects/items/devices/powersink.dm (hunk 2 above)
Comment (ThatLing) [reply yogstation/code/game/objects/items/devices/powersink.dm]: See above

File: code/game/objects/items/devices/powersink.dm (hunk 1 above)
Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [reply code/game/objects/items/devices/powersink.dm]: a watt is one joule per second, this is like saying you have 2 liters per second of water in a bottle

**Comment by ThatLing** [comment]
@1fbff5f83b23d39d38b1dfcb4cac8d9b Changed it

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
thanks love

File: code/game/objects/items/devices/powersink.dm [hunk 3]
```diff
@@ -78,6 +78,10 @@
 				"[user] detaches \the [src] from the cable.", \
 				"<span class='notice'>You detach \the [src] from the cable.</span>",
 				"<span class='italics'>You hear some wires being disconnected from something.</span>")
+	// yogs start - multitool for power drained
+	if(istype(I, /obj/item/multitool))
```
Comment (Amelia0010) [inline code/game/objects/items/devices/powersink.dm]: you can actually modularize this entirely, by using multitool_act() :^)

File: yogstation/code/game/objects/items/devices/powersink.dm [hunk 4]
```diff
@@ -1,4 +1,7 @@
 /obj/item/powersink/examine(mob/user)
 	. = ..()
 	if(isobserver(user))
-		to_chat(user, "The power dial reads [power_drained] J.")
\ No newline at end of file
+		to_chat(user, "The power dial reads [power_drained] J.")
+
+/obj/item/powersink/multitool_act(mob/user)
+	to_chat(user, "The power dial reads [power_drained] J.")
```
Comment (Amelia0010) [inline yogstation/code/game/objects/items/devices/powersink.dm]: if you don't add a Return TRUE it'll also hit the machine, after doing the multitool_act

**Pull request merged**